python/svix/api.py

```diff
@@ -143,18 +143,23 @@ def list(self, options: ApplicationListOptions = ApplicationListOptions()) -> Li
         with self._api() as api:
             return api.list_applications_api_v1_app_get(**options.to_dict(), _check_return_type=False)
 
-    def create(self, application_in: ApplicationIn) -> ApplicationOut:
+    def create(self, application_in: ApplicationIn, idempotency_key: str = "") -> ApplicationOut:
```

Comment:
@svix-frank, should it be a key or a "fetch options" like structure? I think maybe in Python a key makes more sense (option stucts aren't really that common).

Reply:
I think that's reasonable, its how we'll have to do it in other languages so might as well be consistent here.

Reply:
I was actually saying the opposite, that maybe option structures are not that common in Python. Though I'm happy to do an options structure for consistency.